python/hsfs/core/online_ingestion.py

```diff
@@ -91,38 +114,94 @@ def from_response_json(
             return None
 
     def refresh(self):
+        """
+        Refresh the state of this OnlineIngestion object from the backend.
+        """
         online_ingestion = self.feature_group.get_online_ingestion(self._id)
         self.__dict__.update(online_ingestion.__dict__)
 
     def to_dict(self):
+        """
+        Convert the OnlineIngestion object to a dictionary.
+
+        Returns:
+            dict: Dictionary representation of the object.
+        """
         return {"id": self._id, "numEntries": self._num_entries}
 
     def json(self):
+        """
+        Serialize the OnlineIngestion object to a JSON string.
+
+        Returns:
+            str: JSON string representation of the object.
+        """
         return json.dumps(self, cls=util.Encoder)
 
     @property
     def id(self) -> Optional[int]:
+        """
+        Get the unique identifier for the ingestion operation.
+
+        Returns:
+            Optional[int]: The ingestion ID.
+        """
         return self._id
 
     @property
     def num_entries(self) -> Optional[int]:
+        """
+        Get the total number of entries to ingest.
+
+        Returns:
+            Optional[int]: The number of entries.
+        """
         return self._num_entries
 
     @num_entries.setter
     def num_entries(self, num_entries: int) -> None:
+        """
+        Set the total number of entries to ingest.
+
+        Args:
+            num_entries (int): The number of entries.
+        """
         self._num_entries = num_entries
 
     @property
     def results(
         self,
     ) -> List[online_ingestion_result.OnlineIngestionResult]:
+        """
+        Get the list of ingestion results.
+
+        Returns:
+            List[OnlineIngestionResult]: List of ingestion result objects.
+        """
         return self._results
 
     @property
     def feature_group(self) -> fg_mod.FeatureGroup:
+        """
+        Get the feature group associated with this ingestion.
+
+        Returns:
+            FeatureGroup: The associated feature group.
+        """
         return self._feature_group
 
     def wait_for_completion(self, options: Dict[str, Any] = None):
+        """
+        Wait for the online ingestion operation to complete, displaying a progress bar.
+
+        Args:
```

Comment:
```suggestion
        Arguments:
```

This would correct the formatting for the text to resemble the style in other pages